x-pack/plugins/lens/public/trigger_actions/open_lens_config/create_action_helpers.ts

```diff
@@ -52,13 +52,41 @@ export async function executeCreateAction({
   const defaultDataView = await deps.dataViews.getDefaultDataView({
     displayErrors: false,
   });
-  if (!isCompatibleAction || !defaultDataView) {
+
+  const getFallbackDataView = async () => {
```

Comment:
Plan to move this logic to utils when this is merged https://github.com/elastic/kibana/pull/174736